data/json/recipes/nested.json

```diff
@@ -3534,8 +3534,17 @@
     "name": "cardboard boxes",
     "description": "Recipes related to constructing cardboard boxes.",
     "skill_used": "fabrication",
-    "nested_category_data": [ "box_small", "box_medium", "box_large", "box_large_reinforced", "box_oversize" ],
-    "difficulty": 1
+    "nested_category_data": [
```

Comment:
shouldn't there also be a nested recipe for _assembly recipes?

Reply:
My bad, everything is here already. But maybe make two different nests?

Reply:
that good?